document/core/appendix/gen-index-instructions.py

```diff
@@ -75,10 +75,10 @@ def Instruction(name, opcode, type=None, validation=None, execution=None, operat
     Instruction(r'\LOOP~\X{bt}', r'\hex{03}', r'[t_1^\ast] \to [t_2^\ast]', r'valid-loop', r'exec-loop'),
     Instruction(r'\IF~\X{bt}', r'\hex{04}', r'[t_1^\ast~\I32] \to [t_2^\ast]', r'valid-if', r'exec-if'),
     Instruction(r'\ELSE', r'\hex{05}'),
-    Instruction(None, r'\hex{06}'),
-    Instruction(None, r'\hex{07}'),
-    Instruction(None, r'\hex{08}'),
-    Instruction(None, r'\hex{09}'),
+    Instruction(r'\TRY~\X{bt}', r'\hex{06}', r'[t_1^\ast] \to [t_2^\ast]'),
```

Comment:
```suggestion
    Instruction(r'\TRY~\X{bt}', r'\hex{06}', r'[t_1^\ast] \to [t_2^\ast]', r'valid-try-catch', r'exec-try-catch'),
```

Reply:
@rossberg I added these to the `\CATCH` because  `\TRY` could also be the start of a try-delegate instruction. Is there a way to add two validation and two execution links here on the `\TRY`?

Reply:
I see. Yes, it would make sense to have both links there (and remove the ones on catch/delegate. I'd suggest to tweak the Python script to optionally accept a list of validation and execution links.